components/restservice/scripts/creation_validation.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+from os.path import join, dirname
+
+from cloudify import ctx
+
+ctx.download_resource(
+    join('components', 'utils.py'),
+    join(dirname(__file__), 'utils.py'))
+import utils  # NOQA
+
+
+REST_SERVICE_NAME = 'restservice'
+IMMUTABLE_PROPERTIES = [
+    'insecure_endpoints_disabled'
```

Comment:
Upgrade doesn't support 'ssl_enabled'. use it instead of 'insecure_endpoints_disabled'